cloudbridge/providers/azure/azure_client.py

```diff
@@ -1,28 +1,27 @@
 import datetime
 import logging
-from io import BytesIO
+
+import tenacity
+from cloudbridge.interfaces.exceptions import (DuplicateResourceException,
```

Comment:
This was my automatic formatter, should I change it back?

Reply:
This if fine if flake8 is fine with it. Please try running `tox -e lint`.